README.md

```diff
@@ -41,59 +41,83 @@ These [values are what we value](http://www.slideshare.net/reed2001/culture-1798
+* Focus on results, not inputs
+* Take responsibility for understanding context and goals
+
 You can think of this value in two parts: “go get” and “results”. The first part speaks to our strong bias toward action. If there’s something you want to make, improve, change, or get rid of -- go do it. Need input or feedback? Chat with some colleagues on Slack. Need copywriting, engineering or design resources? Recruit those colleagues by rallying them to your cause. Whatever you do, avoid what [Netflix calls](http://www.slideshare.net/reed2001/culture-1798664/11-11_Impact_You_accomplish_amazing) “analysis paralysis”.
 
 The second part (“results”) speaks to the primacy of outputs. We don’t particularly value inputs - how the work gets done. What matters is getting the work done and measuring the impact of this work after completing it. Do the results match up with your goals? Is there more room for improvement? Were there unintended consequences (good or bad)?
 
-Share the results from your work (see our values of “Communicate” and “Default to Transparency”): if they’re great, you’ll be swimming in bonuses. If they weren’t what you hoped for, no big deal - get some input from the team and then keep executing until the results are great.
-
-### Communicate
-We hire team members who we believe will make us smarter and better at what we're trying to do. In order to succeed at becoming smarter and better as a whole, it's essential for us to communicate.
+In practice, this is challenging. In order to go get results, we must understand current goals and priorities, and cost/benefit tradeoffs (ie. go for the quick win that costs little, or spend more time on a robust solution?). Our core value of "Default to Transparency" should ensure that the information you need is available to you, but if it isn't, go seek it out.
 
-What does communication mean at Bonusly?
+### Be an excellent teammate
+* Be proactive and assertive
+* Solicit feedback and input
+* Be [radically candid](https://www.radicalcandor.com/about-radical-candor/)
+* Listen first
+* Assume the best
+* Be there for others
+* Teach & encourage
+* Solve problems together
 
-* You actively solicit input on work you’re doing.
-* You listen before giving your opinion.
-* You contribute your ideas.
-* You keep your cool in stressful interactions.
-* You treat your colleagues with respect at all times, especially times of disagreement.
-* You have the courage to contribute on topics, even if they are not in your realm of responsibilities or expertise.
+We believe that success is achieved by people working on teams. As such, it is critical that we be the best teammates possible to one another. This means being proactive and assertive in your communication: asking for what you need, providing candid, constructive feedback, and voicing your opinions and ideas. It also means listening to, supporting, and encouraging your colleagues -- especially at times of disagreement and stress.
 
```

Comment:
Maybe also add something in this value about, "Sometimes this also means making work fun for your teammates."?